modules/regional-backend-security-policy/variables.tf

```
@@ -0,0 +1,223 @@
+# Copyright 2025 Google LLC
+# Copyright 2025 METRO Digital GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+  description = "Google Cloud project ID."
+  type        = string
+  nullable    = false
+
+  validation {
+    condition     = can(regex("^[a-z][a-z0-9-]{4,30}[a-z0-9]", var.project_id))
+    error_message = <<-EOM
+      It must be 6 to 30 lowercase letters, digits, or hyphens. It must start with a letter.
+      Trailing hyphens are prohibited.
+    EOM
+  }
+}
+
+variable "region" {
+  description = "The region in which security policy is created"
```

Nitpick: To be compliant with style of other descriptions:

```suggestion
  description = "Region in which security policy is created."
```